bundle/src/main/java/com/adobe/acs/commons/widgets/package-info.java

```diff
@@ -20,5 +20,5 @@
 /**
  * JSP Functions for working with custom widgets.
  */
-@org.osgi.annotation.versioning.Version("1.1.1")
+@org.osgi.annotation.versioning.Version("1.2.0")
```

Comment:
A test update shouldnt be changing package export versions! :(

Reply:
I agree, but otherwise we get `[ERROR] com.adobe.acs.commons.widgets: Version increase required; detected 1.1.1, suggested 1.2.0` on build.